### DISCUSSION TITLE
Feature request: `run` work on current line or in visual mode

### DISCUSSION THREAD

**Discussion by desdic** [mistweaverco/kulala.nvim#170] [Q&A] [answered]
If I have a file containing multiple request like

```
GET https://example.com/users

GET https://example.com/users/keys
```

run always takes the first request. A nice feature would be that it runs the request where the cursor is or if its marked visiually.

**Comment by gorillamoe** [comment, marked as answer]
It'll always run the request which is the nearest to the cursor.

You need to separate these requests with `###` like so:

```http
GET https://example.com/users

###

GET https://example.com/users/keys
```

**Reply by desdic** [reply]
Sorry I must have missed that in the documentation .. thanks you :)

**Reply by gorillamoe** [reply]
No worries, that's what the issues are for ❤️ thanks for giving kulala a try!
